loama/src/components/PodList.vue

```diff
@@ -1,31 +1,7 @@
 <template>
     <div>
         <h2>Pods:</h2>
-        <template v-for="pod in pods" :key="pod.pod">
```

Comment:
replacement coming?

Reply:
Yeah, the replacement is this issue: #33